### DISCUSSION TITLE
How to run voskjs with internal microphone?

### DISCUSSION THREAD

**Discussion by MizhgunMenk** [solyarisoftware/voskJs#8] [Q&A]
I want to use it without http. Only like node server. How can i do it?

**Comment by solyarisoftware** [comment]
Could you please clarify what you mean by 'without http" / 'node server'?
Anyway see main project example about how to use audio streaming input from a mic:
https://github.com/alphacep/vosk-api/blob/master/nodejs/demo/test_microphone.js
